### PR TITLE
Prepare for 1.20.0 release

### PR DESCRIPTION
**Pull request by breedx-splk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,17 @@ and this repository adheres to [Semantic Versioning](https://semver.org/spec/v2.
...
 ## Unreleased
 
+## v1.20.0 - 2023-01-17
+
+Regular maintenance release, coordinated after the upstream/vanilla release.
+
+- OpenTelemetry Java SDK updated to version 1.22.0
+- OpenTelemetry Instrumentation for Java dependencies updated to version 1.22.1.
+- Upgrade to Gradle 7.6 (#1062)
+- Upgrade ot Micrometer 1.10.3 (#1065)
```
Comment (mateuszrzeszutek) [inline CHANGELOG.md:18]:
```suggestion
- Upgrade to Micrometer 1.10.3 (#1065)
```

**Review by mateuszrzeszutek — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -162,7 +157,7 @@ To extend the instrumentation with the OpenTelemetry Instrumentation for Java,
 you have to use a compatible API version.
 
 <!-- IMPORTANT: do not change comments or break those lines below -->
-The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.19.0<!--SPLUNK_VERSION--> is compatible
+The Splunk Distribution of OpenTelemetry Java version <!--SPLUNK_VERSION-->1.20.0<!--SPLUNK_VERSION--> is compatible
 with:
 
 * OpenTelemetry API version <!--OTEL_VERSION-->1.21.0<!--OTEL_VERSION-->
```
Comment (mateuszrzeszutek) [inline README.md:163]: Hold on, did you run the pre-release script with all options filled? (`--otel`, `--otel-instrumentation`)
It should bump these versions too

Comment (breedx-splk) [reply README.md:163]: Nope. I'll do that and get the instructions updated.

**Review by theletterf — APPROVED** [review]
Will document the truncate thingie.

**Pull request merged**